### PR TITLE
Document component

### PR DESCRIPTION
**Pull request by summer-cook**
# Summary
Creates the document component for the single request page

# Related
https://github.com/scientist-softserv/webstore-component-library/issues/6

# Screenshots

<details>
<summary>Single Document Item</summary>
image
</details>

<details>
<summary>Document Details</summary>
image
</details>

<details>
<summary>Responsive (smaller Breakpoint)</summary>
image
</details>

<details>
<summary>Separate Line Items Table Component</summary>
image
</details>

# Video
https://share.getcloudapp.com/mXubneE5

### CODE REVIEW

**Comment by summer-cook** [comment]
I just added a commit that updates the documenttype badge to display FIRST on the document component. I think this makes more sense from a user/reading perspective than starting with the ID

image

File: src/components/LineItemsTable/LineItemsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Table from 'react-bootstrap/Table';
+
+const LineItemsTable = (props) => {
+  const{ lineItems, shippingPrice, subtotalPrice, taxAmount, totalPrice } = props
```
Comment (alishaevn) [inline src/components/LineItemsTable/LineItemsTable.jsx:6]: why'd you decide to deconstruct these in the component instead of as props?

Comment (alishaevn) [reply src/components/LineItemsTable/LineItemsTable.jsx:6]: can you run lint on the changed files in this pr please?

File: src/compounds/Document/Document.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        "retail_subtotal_price_currency": "$2.00",
+        "retail_subtotal_price": "2.0",
+        "quantity": "1.0",
+        "currency": "USD",
+        "currency_unit": "$",
+        "name": "Milestone 2",
+        "subtotal": "$2.00",
+        "unit_price": "$2.00"
+    }
+  ],
```
Comment (alishaevn) [inline src/compounds/Document/Document.stories.jsx:46]: can you create this as a variable in args.js and import it back here and in the line items table instead?

File: src/compounds/Document/Document.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    "zipcode": "92075",
+    "country": "US",
+    "latitude": null,
+    "longitude": null,
+    "name": null,
+    "attention": "",
+    "person_name": "",
+    "care_of": null,
+    "text": "505 Lomas Sante Fe Dr\nSuite 110\nSolana Beach, CA, 92075\nUnited States"
+  },
```
Comment (alishaevn) [inline src/compounds/Document/Document.stories.jsx:70]: can you create this and the "shipFrom" object as variables in "args.js" too and then use here? could you also update the Reequest stats story to use this variable? (only the "text" property though)

**Review by alishaevn — CHANGES_REQUESTED** [review]
it looks really good! just a few suggestions and requests.

**Review by alishaevn — APPROVED** [review]

**Pull request merged**